crates/gpui/src/platform/linux/wayland/client.rs

```diff
@@ -273,8 +273,8 @@ impl Client for WaylandClient {
             CursorStyle::IBeam => "text".to_string(),
             CursorStyle::Crosshair => "crosshair".to_string(),
             CursorStyle::ClosedHand => "grabbing".to_string(),
-            CursorStyle::OpenHand => "openhand".to_string(),
-            CursorStyle::PointingHand => "hand".to_string(),
+            CursorStyle::OpenHand => "grab".to_string(),
```

Comment:
Can we annotate this function with the source of these strings?

Reply:
Also, rather than every branch having a `to_string()`, we should call to_string below on line 293.

Reply:
The commit in adwaita-icon-theme link to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor), which itself is based on the [web spec](https://drafts.csswg.org/css-ui/#cursor). I think it makes sense to move the matching to an helper function in `CursorStyle` since it might be used more than once and will be useful for the future web backend. That's also what `winit` does it uses `cursor-icon` which just provides to and from conversions, and also [fallback Xcursor names](https://github.com/rust-windowing/cursor-icon/blob/main/src/lib.rs#L330) which we might want to do as well.

Reply:
Added annotation and fixed `to_string()`. @bbb651 `cursor-icon` package seems really nice. In future, I will see if I can use it for fallback mechanism by using the alt-names from it.